internal/install/command.go

```diff
@@ -21,6 +21,7 @@ var (
 	recipeNames  []string
 	recipePaths  []string
 	testMode     bool
+	entityTags   []string
```

Comment:
@NRhzhao I would have simple name this tags, as those could apply to various things, maybe not entity in the sense of NR1 entity.

Reply:
Thanks @Julien4218  Will make the update